src/audit-logs/audit-logs.ts

```diff
@@ -0,0 +1,16 @@
+import { WorkOS } from '../workos';
+import { AuditLogEventOptions } from './interfaces';
+
+export class AuditLogs {
+  constructor(private readonly workos: WorkOS) {}
+
+  async createEvent(
+    organizationId: string,
```

Comment:
Seems like in our other calls we put `organization_id` as a param inside the options